config/tsconfig.json

```diff
@@ -16,6 +16,7 @@
         "skipLibCheck": true,
         "strict": true,
         "strictNullChecks": false,
-        "suppressImplicitAnyIndexErrors": true
+        "suppressImplicitAnyIndexErrors": true,
+        "strictFunctionTypes": false
```

Comment:
nit: maybe keep the `strict*` stuff together if you're making more changes in this branch?

Reply:
moved